astropy/nddata/tests/test_ccddata.py

```diff
@@ -21,53 +20,16 @@
 from astropy.nddata.ccddata import CCDData
 from astropy.table import Table
 
-# If additional pytest markers are defined the key in the dictionary below
-# should be the name of the marker.
-DEFAULTS = {
-    'seed': 123,
-    'data_size': 100,
-    'data_scale': 1.0,
-    'data_mean': 0.0
-}
 
-DEFAULT_SEED = 123
-DEFAULT_DATA_SIZE = 100
```

Comment:
What about leaving this in and using it [here](https://github.com/astropy/astropy/pull/8889/files#diff-4843b84312ef825592afac740654d93fR25) and then in the test [here](https://github.com/astropy/astropy/pull/8889/files#diff-4843b84312ef825592afac740654d93fR63)? That would avoid hard coding the number 100 in a couple of places.

Reply:
Ah that's a great idea. I added a commit to do exactly that. Thank you!